app/src/main/java/com/keylesspalace/tusky/util/EmojiCompatFont.java

```diff
@@ -158,6 +219,90 @@ public void downloadFont(Context context, Downloader.EmojiDownloadListener... li
         }
     }
 
+    /**
+     * Deletes any older version of a font
+     * @param context The current Context
+     */
+    private void deleteOldVersions(Context context) {
+        File directory = new File(context.getExternalFilesDir(null), DIRECTORY);
+        // It will search for old versions using a regex that matches the font's name plus
+        // (if present) a version code. No version code will be regarded as version 0.
+        Pattern fontRegex = Pattern.compile(getName() + "(\\d+(.\\d+)*)?" + ".ttf");
+
+        if(directory.listFiles() != null) {
+            for (File file : directory.listFiles()) {
```

Comment:
I would call `directory.listFiles()` only once here, e.g store the result in a variable and reuse it

Reply:
Well, it shouldn't really matter, since this shouldn't take too long, but yeah, why not ^^